schema/deploy/shipping/views.sql

```diff
@@ -3316,3 +3318,83 @@ grant select
   to "uw-priority-queue-processor";
 
 commit;
+
+
+create or replace view shipping.linelist_data_for_wa_doh_v1 as (
+    select distinct on (sample_id)
+          -- The naming scheme of these columns is an artefact of how we
+          -- originally submitted data to WaDoH from REDCap reports. Now they
+          -- expect columns structured and named in a specific way, and we
+          -- cannot modify those with our current linelist format. We can notify
+          -- WaDoH of our intent to change our submitted data format (e.g.
+          -- custom format or ELFF), but they'll need sufficient advanced
+          -- notice. At that point, we can update this view with different
+          -- column names.
+          --
+          -- kfay, 5 January 2021
+          sample_id,
+          sampleid.barcode as sample_barcode,
+          collectionid.barcode as collection_barcode,
+          sample.details ->> 'clia_barcode' as scan_id,
+          case
+              when present then 'positive'
+              when present = 'f' then 'negative'
```

Comment:
Could we make these consistent, like
when present = 't' then 'positive'
when present = 'f' then 'negative'

or
when present then 'positive'
when not present then 'negative'

I had to try out the combinations to see how postgres works. I like being explicit with the 't' and 'f' approach. But I had to try that out to see that that equates to true and false.

Reply:
Updated in the latest commit

Reply:
Thanks for that.